polaris/polaris/fee/views.py

```diff
@@ -21,8 +21,8 @@ def _op_type_is_valid(asset_code: str, operation: str, op_type: str) -> bool:
     return False
 
 
-@validate_sep10_token()
 @api_view()
+@validate_sep10_token()
```

Comment:
Does the order of these matter?

Reply:
Unfortunately yes. I tested and if `@validate_sep10_token` was above `@api_view`, the endpoint wouldn't be able to render properly.